components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java

```diff
@@ -312,7 +304,25 @@ protected String getUserStoreDomain(AuthenticatedUser authenticatedUser) throws
         return userStoreDomain;
     }
 
-    private OAuth2AccessTokenRespDTO generateNewAccessTokenResponse(OAuthTokenReqMessageContext tokReqMsgCtx, String scope,
+    private OAuth2AccessTokenRespDTO renewAccessToken(OAuthTokenReqMessageContext tokReqMsgCtx,
+                                                      String scope, String consumerKey, AccessTokenDO existingTokenBean, OauthTokenIssuer oauthTokenIssuer)
+            throws IdentityOAuth2Exception {
+        revokeExistingToken(tokReqMsgCtx.getOauth2AccessTokenReqDTO().getoAuthClientAuthnContext(),
+                existingTokenBean.getAccessToken());
+        // Passing existingTokenBean as null since it is already revoked
+        return generateNewAccessToken(tokReqMsgCtx, scope, consumerKey, null,
+                oauthTokenIssuer);
+    }
+
+    private OAuth2AccessTokenRespDTO issueExistingAccessToken(OAuthTokenReqMessageContext tokReqMsgCtx,
+                                                              String scope, long expireTime, AccessTokenDO existingTokenBean)
+            throws IdentityOAuth2Exception {
```

Comment:
Add a blank line after method declaration. Fix it in the other places too

Reply:
Fixed